docs/settings.rst

```diff
@@ -1035,6 +1035,11 @@ You can use the following settings to configure the pagination.
    The maximum number of articles to include on a page, not including orphans.
    False to disable pagination.
 
+.. data:: MAX_PAGINATION = False
+
+   The maximum number of pages generated, not including orphans.
+   False to paginate all available pages.
```

Comment:
Orphans? They don't affect this.

Also, `False` should be codified: ``` ``False`` ```

Reply:
I was copy-pasting the above line. The concept of orphan pages is a bit vague for me but I can remove the reference if better.

Reply:
Let's say, you have 31 articles with pagination 10. You'd end up with 4 pages where the last page would only contain 1 article. Orphans decide how many articles should be in that last page minimum. If it ends up below that threshold, last two pages are combined. So instead of 4 pages (10+10+10+1), you can get 3 pages (10+10+11).